components/brave_rewards/resources/ui/components/pageWallet.tsx

```diff
@@ -70,16 +70,36 @@ class PageWallet extends React.Component<Props, State> {
   }
 
   onModalBackupOnCopy = (backupKey: string) => {
-    clipboardCopy(backupKey)
+    const success = clipboardCopy(backupKey)
+    // TODO(jsadler) possibly flash a message that copy was completed
+    console.log(success ? 'Copy successful' : 'Copy failed')
   }
 
   onModalBackupOnPrint = () => {
     // TODO NZ implement
     console.log('onModalBackupOnPrint')
   }
 
+  constructBackup = (backupKey: string) => {
+    const backupString = getLocale('backupFileText1') + '\n' +
```

Comment:
this needs to be hard coded to english, so that we have always the same export file for the import

Reply:
Will do

Reply:
Done.